packages/collector-base/src/propsRegistry.ts

```diff
@@ -2,23 +2,33 @@ import { creatFilePropsKey, createFilePropList } from './createFilePropList';
 
 export const propsRegistry = new Map();
 
-export type AddPropsFn = (
-  filePath: string,
-  exportName: string,
-  propSet: Record<string, any>,
-  name?: string
-) => void;
+export type PropSetType = 'json' | 'js';
```

Comment:
whats the diff to an enum?

Reply:
`enum` is like `PropSetType.json`.
This one is just a plain string but it is validated with TypeScript.